packages/api/server/http.mts

```diff
@@ -45,6 +47,37 @@ router.use(express.json());
 
 router.options('/file', cors());
 
+/**
+ * Note things added with data.append will be 2: in stream, LLM res is 0: so you can
```

Comment:
I don't understand this comment. Can we reword it?

Reply:
let me think of a good way to phrase it but essentially the streams look like this

```
2:["initialized call"]
0:"To"
0:" store a user ID,"
0:" times of creation and update"
0:", email, and username"
0:", you can use a"
0:" database or"
0:" a data"
0:" structure"
0:" depending"
0:" on your specific needs"
0:". Here's an example"
2:["random interject", "with more data"]
0:" of"
0:" how you might"
0:" structure"
0:" this data in different"
0:" formats"
0:":\n\n1"
0:". SQL"
0:" Database Table"
0:":\n\n```sql"
0:"\nCREATE TABLE users ("
0:"\n    i"
```

in here only 0:"<content here>" will be in the display stream while 2:[...] will just be data sent down to trigger things